run_create_s3_bucket.py

```diff
@@ -73,6 +73,35 @@ def run_create_s3_bucket(name, settings):
         aws_cli.run(cmd)
 
     ################################################################################
+    if 'hbsmith-script' in bucket_name:
+        cmd = ['s3api', 'get-bucket-versioning']
+        cmd += ['--bucket', bucket_name]
+        rr = aws_cli.run(cmd)
+
+        if rr and rr['Status'] == 'Enabled':
+            print_message('set script bucket life cycle rule')
+            cc = {
+                "Rules": [
+                    {
+                        "ID": "script_file_manage_rule",
+                        "Status": "Enabled",
+                        "Filter": {
+                        },
+                        "NoncurrentVersionExpiration": {
+                            "NoncurrentDays": expire_days
```

Comment:
@GoldieKMG // 위에서 의견 드렸지만 만일의 경우를 대비하기 위해 아예 다른 변수를 명시적으로 선언해서 쓰는걸 제안드립니다.
```suggestion
                            "NoncurrentDays": versioning_expire_days
```